identity/handler_test.go

```diff
@@ -73,7 +73,9 @@ func TestHandler(t *testing.T) {
 
 	var send = func(t *testing.T, base *httptest.Server, method, href string, expectCode int, send interface{}) gjson.Result {
 		var b bytes.Buffer
-		require.NoError(t, json.NewEncoder(&b).Encode(send))
+		if send != nil {
+                        require.NoError(t, json.NewEncoder(&b).Encode(send))
+                }
 		req, err := http.NewRequest(method, base.URL+href, &b)
```

Comment:
Here, **&b** value will remain nil by default. That's why when the **send** value is nil, we are sending http request body value as nil. This will create io.EOF error in update() function.